docs/readthedocs/source/doc/PPML/Overview/ppml.md

```diff
@@ -1,63 +1,63 @@
-# PPML User Guide
+# Privacy Preserving Machine Learning (PPML) User Guide
 
-## 1. Privacy Preserving Machine Learning
-Protecting privacy and confidentiality is critical for large-scale data analysis and machine learning. BigDL ***PPML*** combines various low level hardware and software security technologies (e.g., Intel SGX, LibOS such as Graphene and Occlum, Federated Learning, etc.), so that users can continue to apply standard Big Data and AI technologies (such as Apache Spark, Apache Flink, Tensorflow, PyTorch, etc.) without sacrificing privacy.
+## 1. Introduction
+Protecting privacy and confidentiality is critical for large-scale data analysis and machine learning. BigDL ***PPML*** combines various low-level hardware and software security technologies (e.g., [Intel® Software Guard Extensions (Intel® SGX)](https://www.intel.com/content/www/us/en/architecture-and-technology/software-guard-extensions.html#:~:text=Intel%C2%AE%20SGX%20allows%20user,level%20of%20control%20and%20protection.), [Library Operating System (LibOS)](https://events19.linuxfoundation.org/wp-content/uploads/2017/12/Library-OS-is-the-New-Container-Why-is-Library-OS-A-Better-Option-for-Compatibility-and-Sandboxing-Chia-Che-Tsai-UC-Berkeley.pdf) such as [Graphene](https://www.usenix.org/conference/atc17/technical-sessions/presentation/tsai) and [Occlum](https://github.com/occlum/occlum), Federated Learning, etc.), so that users can continue to apply standard Big Data and AI technologies (such as Apache Spark, Apache Flink, Tensorflow, PyTorch, etc.) without sacrificing privacy.

```

Comment:
(https://www.intel.com/content/www/us/en/architecture-and-technology/software-guard-extensions.html) is enough. Remove other parameters

Reply:
[Graphene](https://github.com/gramineproject/graphene)